core/src/main/scala/org/apache/spark/h2o/backends/internal/InternalH2OBackend.scala

```diff
@@ -64,6 +64,10 @@ class InternalH2OBackend(@transient val hc: H2OContext) extends SparklingBackend
       conf.setCloudName("sparkling-water-" + System.getProperty("user.name", "cluster") + "_" + conf.sparkConf.getAppId)
     }
 
+    if (AzureDatabricksUtils.isRunningOnAzureDatabricks(conf)) {
+      conf.setClientCheckRetryTimeout(600000)
```

Comment:
Is not it dangerous to force it? Better would be to use this path on Azure only if user does not specify the config property manually (use-case: imagine `600000` is not enough - how can user bump it  to `100000`?)

Also nit: i would put some message to log that we do not forget about it .

Reply:
Thanks, good suggestions!